nimbus-consensus/src/import_queue.rs

```diff
@@ -1,18 +1,18 @@
-// Copyright 2019 Parity Technologies (UK) Ltd.
-// This file is part of Cumulus.
+// Copyright 2019-2021 PureStake Inc.
+// This file is part of Moonbeam.
```

Comment:
```suggestion
// This file is part of Nimbus.
```

Reply:
I don't have strong feelings here, but I think libraries usually use their own name here as opposed to the flagship product that uses them.

Reply:
done